### PR TITLE
Update cookie macros about links styled as buttons

### PR DESCRIPTION
**Pull request by EoinShaughnessy**

### CODE REVIEW

**Review by vanitabarrett — COMMENTED** [review]
👍 Looks good to me, but I'll let @hannalaakso take a final look as she's been working on those changes

**Comment by EoinShaughnessy** [comment]
> 👍 Looks good to me, but I'll let @hannalaakso take a final look as she's been working on those changes

@vanitabarrett Thanks! Yeah, got the nod from @hannalaakso earlier.

**Review by TechPunk316 — COMMENTED** [review]
Looks ok to me

**Review by SteveEvansGDS — APPROVED** [review]

**Comment by EoinShaughnessy** [comment]
This content has passed 2i.

**Comment by EoinShaughnessy** [comment]
@hannalaakso I've added the Changelog update to this PR. Should I squash the 2 commits here, or would it better to leave them be, because they represent changes to 2 different docs (the cookie macros and the Changelog)?

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,16 @@
...
 ## New features
 
-- [#2164: Enable cookie banner to set link styled as a button](https://github.com/alphagov/govuk-frontend/pull/2164)
+#### Allow users to use cookie banner buttons as links
+
+You can now add links styled as buttons to cookie banners.
+
+Use this feature only if you need to. For example, to display the **Hide** button because:
+
+- your cookie banner needs to work without JavaScript
```
Comment (EoinShaughnessy) [inline CHANGELOG.md:13]: Does the **Hide** button not display for non-JS users unless they style links as buttons?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,7 +4,16 @@
...
+- the user has already accepted or rejected cookies - so, when they press **Hide** and the page reloads, the banner is hidden
+
+This was added in [pull request #2164: Enable cookie banner to set link styled as a button](https://github.com/alphagov/govuk-frontend/pull/2164)
```
Comment (EoinShaughnessy) [inline CHANGELOG.md:16]: 'Link' makes me think of navigation to another webpage. What, exactly, do we mean by 'link' in the context of the cookie banner?

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,7 +4,16 @@
 
 ## New features
 
-- [#2164: Enable cookie banner to set link styled as a button](https://github.com/alphagov/govuk-frontend/pull/2164)
+#### Allow users to use cookie banner buttons as links
+
+You can now add links styled as buttons to cookie banners.
+
+Use this feature only if you need to. For example, to display the **Hide** button because:
```
Comment (EoinShaughnessy) [inline CHANGELOG.md:11]:
```suggestion
Use this feature only if you need to. For example, you can set the **Hide** button to be a link styled as a button that reloads the page. You might want to do this if the cookie banner needs to work without JavaScript and the **Hide** button does not need to set a cookie, as the user has already accepted or rejected cookies.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,7 +4,16 @@
 
 ## New features
 
-- [#2164: Enable cookie banner to set link styled as a button](https://github.com/alphagov/govuk-frontend/pull/2164)
+#### Allow users to use cookie banner buttons as links
+
+You can now add links styled as buttons to cookie banners.
+
+Use this feature only if you need to. For example, you can set the **Hide** button to be a link styled as a button that reloads the page. You might want to do this if the cookie banner needs to work without JavaScript and the **Hide** button does not need to set a cookie, as the user has already accepted or rejected cookies.
+
```
Comment (EoinShaughnessy) [inline CHANGELOG.md:12]:
```suggestion
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,7 +4,16 @@
...
+- your cookie banner needs to work without JavaScript
```
Comment (EoinShaughnessy) [inline CHANGELOG.md:12]:
```suggestion
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,7 +4,15 @@
...
 ## New features
 
-- [#2164: Enable cookie banner to set link styled as a button](https://github.com/alphagov/govuk-frontend/pull/2164)
+#### Allow users to use cookie banner buttons as links
+
+You can now add links styled as buttons to cookie banners.
+
+Use this feature only if you need to. For example, you can set the **Hide** button to be a link styled as a button that reloads the page. You might want to do this if the cookie banner needs to work without JavaScript and the **Hide** button does not need to set a cookie, as the user has already accepted or rejected cookies.
+- your cookie banner needs to work without JavaScript
+- the user has already accepted or rejected cookies - so, when they press **Hide** and the page reloads, the banner is hidden
```
Comment (EoinShaughnessy) [inline CHANGELOG.md:12]:
```suggestion

```

File: CHANGELOG.md [hunk 7]
```diff
@@ -4,7 +4,14 @@
 
 ## New features
 
-- [#2164: Enable cookie banner to set link styled as a button](https://github.com/alphagov/govuk-frontend/pull/2164)
+#### Allow users to use cookie banner buttons as links
+
+You can now add links styled as buttons to cookie banners.
+
+Use this feature only if you need to. For example, you can set the **Hide** button to be a link styled as a button that reloads the page. You might want to do this if the cookie banner needs to work without JavaScript and the **Hide** button does not need to set a cookie, as the user has already accepted or rejected cookies.
+- the user has already accepted or rejected cookies - so, when they press **Hide** and the page reloads, the banner is hidden
```
Comment (EoinShaughnessy) [inline CHANGELOG.md]:
```suggestion

```

**Comment by EoinShaughnessy** [comment]
@hannalaakso I've reduced the commits to 2 (1 for the Changelog update, 1 for the macros update). Can I get a review? :)

**Review by hannalaakso — APPROVED** [review]
Looks great, thanks @EoinShaughnessy 👍

**Pull request merged**